packages/integrations/markdoc/README.md

```diff
@@ -249,6 +212,83 @@ export default defineMarkdocConfig({
 })
 ```
 
+### Set the root HTML element
+
+Markdoc wraps documents with an `article` tag by default. This can be changed from the `document` Markdoc node. This accepts an HTML element name or `null` if you prefer to remove the wrapper element:
+
+```js
+// markdoc.config.mjs
+import { defineMarkdocConfig, nodes } from '@astrojs/markdoc/config';
+
+export default defineMarkdocConfig({
+  nodes: {
+    document: {
+      render: null,
```

Comment:
Can we show an HTML element here instead? I feel like `null` is the easier one to get correct. But, some might try `h2` and others `'h2'`, some might try `<h2>`...

Or, we could do something like:

```
render: null, // default 'article'
```
(Admittedly I wasn't sure exactly the syntax to use here! I assumed string?)